src/test/librbd/test_support.cc

```diff
@@ -22,6 +22,33 @@ bool is_feature_enabled(uint64_t feature) {
   return (get_features(&features) && (features & feature) == feature);
 }
 
+int create_image_full_pp(librbd::RBD &rbd, librados::IoCtx &ioctx,
```

Comment:
`create_image_pp()` below is not changed to use this helper as `DiffIterateTest/1.DiffIterate` test case fails with `stripingv2` image.